src/style.css

```diff
@@ -1 +1,25 @@
 /* styles go here */
+label {
+  display: block;
+}
+
+.personal_information_label {
```

Comment:
Consider using a more generic class name for the label, such as 'form-label' instead of 'personal_information_label', as this can be used for other forms as well. 🌟